.devcontainer/devcontainer.json

```
@@ -52,7 +52,8 @@
 	"remoteUser": "vscode",
 
 	"features": {
-        "ghcr.io/devcontainers/features/docker-outside-of-docker:1": {},
+        "ghcr.io/devcontainers/features/sshd:1": {"version": "latest"},
```

```suggestion
        "ghcr.io/devcontainers/features/sshd:1": {},
```
"version" defaults to "latest", we don't specify it anywhere else, no reason to break the pattern